### ISSUE TITLE
Problem Unknown (All Downloads Failing)

### ISSUE THREAD

**Issue by Damo1977** [Ascoware/get-iplayer-automator#66]
*What steps will reproduce the problem? Be specific, and provide as much detail as possible.*
All BBC and ITV programs are failing to download. Whether selected from the cache or current webpage function.

*What is the expected output? What do you see instead?*
Get a failed download solution window which says Problem Unknown

*What version of GiA are you using? On what version of OS X?*
Version 1.9.7 (20171007004). Mac OSX High Sierra v 10.13

*Are you using a web proxy? DNS proxy? VPN? VPS?*
No, but have tried it via VPN and had same issue. I'm in the UK with BT as my ISP.

*What program are you attempting to download (full title, series and episode)? **Provide only ONE program as an example.** *
Eastenders 12/10/2017 is failing as a test.

*Please provide any additional information below.*
Was also having same issue on a previous version of Get Iplayer Automator. Deleted this and the application support folder then updated to 1.9.7. Have also tried to download whilst connected to O2 as my ISP and it still failed.

**Comment by Lordsnooty05** [comment]
Damo, have you deleted the history? Sometimes I would have got that where the program had failed to download and then it was in my history and once I deleted all the programs and then shut down GIA and then open it up it would download correctly. At the minute this version of GIA is working perfectly for me and is downloading ITV and BBC programs......previous versions had problems with ITV but now both channels are working for me

**Comment by Damo1977** [comment]
Thanks Lordsnooty. I tried deleting the history but no joy. However I've since found out that my Mac Server had stopped. Although I was running Get Iplayer Automator on my MacBook, the files were being saved to my networked downloads folder which wasn't available. Having restarted my server the location is now available and the downloads are running fine.

Apologies for raising this, I've now closed the issue.
